src/cargo/core/compiler/build_context/target_info.rs

```diff
@@ -697,10 +700,13 @@ impl<'cfg> RustcTargetData<'cfg> {
         // `--target` flag is not specified. Since the unit_dependency code
         // needs access to the target config data, create a copy so that it
         // can be found. See `rebuild_unit_graph_shared` for why this is done.
-        if requested_kinds.iter().any(CompileKind::is_host) {
+        let is_cross = if requested_kinds.iter().any(CompileKind::is_host) {
```

Comment:
Minor nit: I'd suggest putting `let is_cross = !requested_kinds.iter().any(CompileKind::is_host)` on a previous line, and then changing the `if` to just be `if !is_cross`.

Reply:
applied